PDBminer

```diff
@@ -651,34 +653,48 @@ def alignment(uniprot_sequence, AA_pdb):
     pdb_sequence = pdb_sequence.replace("-", "X" )
     
     alignments = aligner.align(uniprot_sequence, pdb_sequence)
-    #if there are gaps: sequence contains letters not in the alphabet
-    #"-" is not understood. 
     
-    if alignments.score < 5: 
-        uniprot_aligned = []
-        pdb_aligned = []
-        modified_string = []
-    else:
-        a = format(alignments[0])
-        a = a.split("\n")    
-        
-        uniprot_aligned = a[0].replace(" ", "-")
-        uniprot_aligned = uniprot_aligned.replace("X", "-")
-        pdb_aligned = a[2].replace(" ", "-")
-        pdb_aligned = pdb_aligned.replace("X", "-")
-        
-        positions = a[1].replace(" ", "-")
-        first = positions.find('|')
-        last = positions.rfind('|')
-        modified_string = ''
-        for i in range(len(pdb_aligned)):
-            if i < first or i > last:
-                modified_string += '-'
-            else:
-                modified_string += pdb_aligned[i]
-        
-    return uniprot_aligned, pdb_aligned, modified_string
+    if alignments.score < peptide_min_length: 
+        return [], []
```

Comment:
maybe you could return `None` instead? It's usually used as a flag value to mean "no results"